packages/config/config/devices/0x0190/adc-t_3000.json

```diff
@@ -0,0 +1,801 @@
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "Battery",
+					"value": 0
+				},
+				{
+					"label": "C-Wire",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "27",
+			"label": "Battery Alert Threshold Low",
+			"description": "Battery Alert Range",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 30
+		},
+		{
+			"#": "28",
+			"label": "Battery Alert Threshold Very Low",
+			"description": "Very Low Battery Alert Range (percentage)",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 15
+		},
+		{
+			"#": "28",
+			"label": "Current Relay State",
+			"description": "Detected loads and current states of on-board relays",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 4294967295,
+			"unsigned": true,
+			"readOnly": true
+		},
+		{
+			"#": "30",
+			"label": "Remote Temperature Enable",
+			"description": "Enables remote temperature sensor instead of built-in.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "31",
+			"label": "Heat Differential",
+			"unit": "0.5°F",
+			"valueSize": 4,
+			"minValue": 2,
+			"maxValue": 20,
+			"defaultValue": 6
+		},
+		{
+			"#": "32",
+			"label": "Cool Differential",
+			"unit": "0.5°F",
+			"valueSize": 4,
+			"minValue": 2,
+			"maxValue": 20,
+			"defaultValue": 6
+		},
+		{
+			"#": "33",
+			"label": "Temperature Reporting Threshold",
+			"unit": "0.5°F",
+			"valueSize": 4,
+			"minValue": 1,
+			"maxValue": 4,
+			"defaultValue": 2
+		},
+		{
+			"#": "35",
+			"label": "Z-Wave Echo Association Reports",
+			"description": "Enable/Disabled Echo Assoc. Reports.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "36",
+			"label": "C-Wire Power Thermistor Offset",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": -10,
+			"maxValue": 10,
+			"defaultValue": -2
+		},
+		{
+			"#": "37",
+			"label": "Run Fan With Auxiliary Heat",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "38",
+			"label": "Z-Wave Association Report Bit Mask",
+			"description": "Bitmask to selectively enable non-required Z-wave association reports.",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 4294967295,
+			"defaultValue": 4294967295,
```

Comment:
Is there a better way to represent bitmasks? I am not specifying the values for each bit here, it would be convenient if there was a way to do that, but didn't find one.

Reply:
See my other [comment](https://github.com/zwave-js/node-zwave-js/pull/4131/files/cbba4e4731a4f9709617fa7e10428eb6aeeb07e4#r792375463).